collectd/configs/facette.json

```diff
@@ -0,0 +1,5 @@
+{
+  "bind": ":80",
+  "providers_dir": "/etc/facette/providers",
+  "pid_file": "/var/run/facette/facette.pid"
```

Comment:
Is this necessary with runit? Processes should stay attached to parent (no forking, no pidfiles, etc)